### PR TITLE
feat(curriculum): add challenges to english block 1

### PR DESCRIPTION
**Pull request by moT01**
This adds all the challenges for the first English block. 

The main thing I want the English team to look at here is what is in backticks. Anything in backticks will not be translated - anything not in backticks will be translated. So make sure that everything you intended to be in backticks is correct (stays in english). 

Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Ref: https://github.com/freeCodeCamp/freeCodeCamp/issues/52400

### CODE REVIEW

**Review by nieldakarla — CHANGES_REQUESTED** [review]
Hey, it is looking good! Thanks for uploading more than 180 challenges! =D

Here are somethings we need to change:
Some dialogues changed after Quincy recorded them. So we'll have to remove:
task 140, 141, 144, 145, 155, 156, 157, 158, 159, 160, 161, 179, 180, 181, 

Some characters changed:

Change Tom to Sarah.
Task 166 , 167, 168, 169, 171, 173, 182

Change Shopie to Jake
175, 176, 177, 178, 178, 182

Dialogue titles to change:
 "Dialogue 5: End of The First Day"

**Comment by moT01** [comment]
Thanks for the review @nieldakarla. There are still a number of tasks in the last dialogue that reference Tom:
Tasks  151, 153, 160, 164, 165, and 166. Can you take a look at those and confirm that we still want to keep it as Tom or let me know if we want to change them to Sarah.

File: curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656d2600b0ffa811dac3853f.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+id: 656d2600b0ffa811dac3853f
+title: Task 151
+challengeType: 22
+dashedName: task-151
+audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
+---
+
+<!--
+AUDIO REFERENCE:
+Jake: Hey. You are Tom, right? I'm Jake, from Security.
```
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656d2600b0ffa811dac3853f.md:11]:
```suggestion
Jake: Hey. You are Sarah, right? I'm Jake, from Security.
``

File: curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656d26bc907ce0123546288a.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+---
+
+<!--
+AUDIO REFERENCE:
+Jake: I'm here to give you your access card.
+-->
+
+# --description--
+
+The phrase `I'm here to` normally talks about purpose or intention. Jake uses this structure to explain why he's talking to Tom.
```
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656d26bc907ce0123546288a.md:16]:
```suggestion
The phrase `I'm here to` normally talks about purpose or intention. Jake uses this structure to explain why he's talking to Sarah.
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656d26bc907ce0123546288a.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+`I'm here to _ you your access card.`
+
+## --blanks--
+
+`give`
+
+### --feedback--
+
+Focus on why Jake is there. It's about providing Tom with something.
```
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656d26bc907ce0123546288a.md:30]:
```suggestion
Focus on why Jake is there. 
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656d2aee945ce2137bd8b8b8.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+challengeType: 19
+dashedName: task-160
+audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
+---
+
+<!--
+AUDIO REFERENCE:
+Tom: Thanks, Jake. Is it contactless?
+Jake: No, it isn't. It's the good-old swipe at the door. When you hear the click, it is unlocked, and you can get in or out.
+Tom: Good to know. Thank you!
```
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656d2aee945ce2137bd8b8b8.md:13]:
```suggestion
Jake: No, it isn't. It's the good-old swipe at the door. When you hear the click, it is unlocked, and you can get in or out.
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656d2c94279eb6143485eac6.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+What does Jake mean about Tom's first day? It's about to end.
+
+---
+
+`it`
+
+### --feedback--
+
+What does Jake mean about Tom's first day? It's about to end.
```
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656d2c94279eb6143485eac6.md:50]:
```suggestion
Demonstrative pronoun

---

`is`

### --feedback--

Verb to be in the third person

---

`it`

### --feedback--

It is a pronoun!
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656d2d19c570e2146b5e8835.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+---
+
+<!--
+AUDIO REFERENCE:
+Jake: Well, five o'clock. I guess this is it for your first day. How was it?
+-->
+
+# --description--
+
+Sometimes, people use phrases to ask about experiences, feelings, or opinions. One such phrase is `How was it?` which is a general way to ask about something that just happened. Jake uses the phrase `How was it?` to ask Tom about his experience. Let's find out what she wants to know.
```
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656d2d19c570e2146b5e8835.md:16]:
```suggestion
Sometimes, people use phrases to ask about experiences, feelings, or opinions. `How was it?` is a general way to ask about something that just happened. 
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656d2d19c570e2146b5e8835.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+
+`How was Tom's first day in the office.`
+
+---
+
+`Where Tom is going.`
+
+### --feedback--
+
+She isn't asking about Tom's location.
```
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656d2d19c570e2146b5e8835.md:50]:
```suggestion
`What Sarah will do tomorrow.`

### --feedback--

He didn't ask about future plans.

---

`How was Sarah's first day in the office.`

---

`Where Sarah is going.`

### --feedback--

He isn't asking about Sarah's location.
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656d2d814b60b6149a03c699.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+title: Task 166
+challengeType: 19
+dashedName: task-166
+audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
+---
+
+<!--
+AUDIO REFERENCE:
+Jake: Well, five o'clock. I guess this is it for your first day. How was it?
+Tom: Good, really good. See you tomorrow, then?
```
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656d2d814b60b6149a03c699.md:12]:
```suggestion
Jake: Well, five o'clock. I guess this is it for your first day. How was it?
Sarah: Good, really good. See you tomorrow, then?
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656d2d814b60b6149a03c699.md [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+
+# --description--
+
+`See you tomorrow` is a casual way of saying goodbye and expressing the hope to meet the person the next day.
+
+# --question--
+
+## --text--
+
+After answering Jake's question, Why does Tom say `See you tomorrow`?
```
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656d2d814b60b6149a03c699.md:23]:
```suggestion
After answering Jake's question, Why does Sarah say `See you tomorrow`?
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656d2d814b60b6149a03c699.md [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+
+## --answers--
+
+`He will meet her now.`
```
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656d2d814b60b6149a03c699.md:27]:
```suggestion
`They will meet now.`
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656d2d814b60b6149a03c699.md [hunk 11]
```diff
@@ -0,0 +1,59 @@
...
+
+`They will never meet again.`
+
+### --feedback--
+
+It's a positive statement.
+
+---
+
+`He hopes to see her tomorrow.`
```
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656d2d814b60b6149a03c699.md:43]:
```suggestion
`She hopes to see him tomorrow.`
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656d2d814b60b6149a03c699.md [hunk 12]
```diff
@@ -0,0 +1,59 @@
...
+
+### --feedback--
+
+This is about meeting again soon.
+
+---
+
+`He doesn't want to see her.`
```
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656d2d814b60b6149a03c699.md:51]:
```suggestion
`She doesn't want to see him.`
```

**Review by nieldakarla — CHANGES_REQUESTED** [review]
Good catch with the challenges that were still using Tom's name. 

I believe I've removed all elements that had to be removed. I've also checked some pronouns that were wrong due to the changes in characters.

**Comment by nieldakarla** [comment]
@moT01 I've just reviewed the challenges you asked. Already created another change request with the changes. =)

File: curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/6568c3539307ad0f902d20e7.md [hunk 13]
```diff
@@ -0,0 +1,54 @@
...
+
+`Toronto`
+
+---
+
+`Vancouver`
+
+### --feedback--
+
+Listen again..
```
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/6568c3539307ad0f902d20e7.md:34]:
```suggestion
Listen again.
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/6568c3539307ad0f902d20e7.md [hunk 14]
```diff
@@ -0,0 +1,54 @@
...
+
+---
+
+`Montreal`
+
+### --feedback--
+
+Listen again..
```
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/6568c3539307ad0f902d20e7.md:42]:
```suggestion
Listen again.
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656cb2ee9d60f205d362b1de.md [hunk 15]
```diff
@@ -0,0 +1,63 @@
...
+
+`Cutting-edge gadgets`
+
+### --feedback--
+
+`Cutting-edge gadgets` is a general term used by Sophie to describe the latest technology or tools. The other options are specific items Tom lists when describing his workspace. Look for the exact items mentioned by Tom.
+
+## --video-solution--
+
+4
```
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656cb2ee9d60f205d362b1de.md:63]:
```suggestion
Which of these is NOT a workplace item mentioned by Sophie and Tom?

## --answers--

`Standing desk`

### --feedback--

Tom mentions this workplace Item.

---

`Ergonomic chair`

### --feedback--

Tom mentions this workplace Item.

---

`Ergonomic mouse`

### --feedback--

Tom mentions this workplace Item.

---

`Cutting-edge gadgets`

## --video-solution--

4
```

I noticed the question in this challenge wasn't specific enough. I've also noticed the feedbacks were wrong. I've fixed that.

**Review by nieldakarla — COMMENTED** [review]
Thanks for making the changes. I removed some extra dots in two options and fixed a challenge so the question was more precise and the feedback appropriate.

**Review by nieldakarla — APPROVED** [review]

**Comment by DanielRosa74** [comment]
Hi @moT01 and @nieldakarla !

Here's the items I found in my review:

https://github.com/freeCodeCamp/freeCodeCamp/pull/52472/files#diff-950cf32ae4808e5921ee91ed78318ea7524dfad7ada08b334a50ef0edc8dc133

First Day at the Office - Task 3

Any chance we can remove "[Name]" and "[Job title]" from the backticks? I believe these should be translatable.

https://github.com/freeCodeCamp/freeCodeCamp/pull/52472/files#diff-7979a4b6935f155a5d48b1811f5978b7b9515649411fb5d57156c5d380a76932

First Day at the Office - Task 6

@nieldakarla, won't there be any description in a fill in the blanks task?

https://github.com/freeCodeCamp/freeCodeCamp/pull/52472/files#diff-035cb6096020efcd49e33f41f168d470c962a8dde866179977884ddcb1a7fa32

First Day at the Office - Task 11

@nieldakarla, should the answers to choose from be in English in this case? It's an interpretation question. The language is simple, though, and that could be a reason to leave it as is. Also, the feedback will be translated, which might help the person understand the content of the answers if they got them wrong.

https://github.com/freeCodeCamp/freeCodeCamp/pull/52472/files#diff-d7eb4c9b0346f9b79ff9a41cd67ed198e7d9ad577731bab9727bdba36d64c9df

First Day at the Office - Task 15

Tom is not from Toronto. He's from Texas now. Mistake in the audio reference and the sentence.

https://github.com/freeCodeCamp/freeCodeCamp/pull/52472/files#diff-8b8e6f0c41f110410d1c722cae8bc768bd3e7170430f8b4d8cde677ad4e56228

First Day at the Office - Task 16

Tom is not from Toronto. He's from Texas now. Mistake in the audio reference and the sentence.

The question has to change to "What is Tom comparing California to?" Probably changing the other answers to US states would be better as well.

https://github.com/freeCodeCamp/freeCodeCamp/pull/52472/files#diff-640ba803ab7ed3c759a625e2339575bad92a9187209ac4c1398ebd76f90d053b

First Day at the Office - Task 17

Tom is not from Toronto. He's from Texas now. Mistake in the audio reference, feedbacks and the sentence.

https://github.com/freeCodeCamp/freeCodeCamp/pull/52472/files#diff-930654f9baa7abf5a7f81c57ca5c36e7cfbce847655bb64a517b338a80efe0eb

First Day at the Office - Task 18

Change Toronto to Texas. Mistake in the audio reference and the sentence.

https://github.com/freeCodeCamp/freeCodeCamp/pull/52472/files#diff-449c9ffa26f64fab60e4544bb1697c48cb7c2c28ad6e2fc7306795b0b1c3bca1

First Day at the Office - Task 19

Change Toronto to Texas. Mistake in the audio reference, description and the sentence. Also in the first answer and in the feedback to the right answer (California).

https://github.com/freeCodeCamp/freeCodeCamp/pull/52472/files#diff-7f1bccb86272ecc42f77433bf501d8fbb68ae2d7f4433b55350a8cc84a78f1b5

First Day at the Office - Task 20

Change Toronto to Texas. Mistake in the audio reference and in one of the answers "He misses Toronto a lot.".

https://github.com/freeCodeCamp/freeCodeCamp/pull/52472/files#diff-d95e4f46913ac579321f7e9cf8350e3578f52b3fb2b24a40f796165a391f9dfd

First Day at the Office - Task 21

Change Toronto to Texas. Mistake in the audio reference and the description.

https://github.com/freeCodeCamp/freeCodeCamp/pull/52472/files#diff-53fc3fee437a004d4785faf060c8ee657dc3220692243132f6b9e4210de683ca

First Day at the Office - Task 22

Could [place] be out of the backticks in the the description?

https://github.com/freeCodeCamp/freeCodeCamp/pull/52472/files#diff-2e3a2ca5b6352b7b9586f36dd4d2904277791de6ec59ee62fc9ef4e295afadb8

First Day at the Office - Task 32

The audio reference (which I believe will be removed) is missing an "around" between "me" and "the".  This reflects in the sentence as well.

https://github.com/freeCodeCamp/freeCodeCamp/pull/52472/files#diff-e873ce25dcbc4ad461a7ad68d3bd7b635caacea18f2d82a8ce4303b46fa54167

First Day at the Office - Task 34

@nieldakarla, same question of task 11. Should the question (which most likely the A2 student is already familiar to) be translated? In this case, more than in 11, I see no reason for change.

https://github.com/freeCodeCamp/freeCodeCamp/pull/52472/files#diff-3c6aeb39793a113c22b2dc43e1415daad602ba6911af78a87c42b0c5416b62ec

First Day at the Office - Task 36

@nieldakarla, same question of task 34. If we keep 34 between backticks, maybe we should do the same here.

https://github.com/freeCodeCamp/freeCodeCamp/pull/52472/files#diff-40a70621cb5e3db4f6c3bee54d5767c41eebc64c7a5ce7daa4e091c305bd4f10

First Day at the Office - Task 38

The example references Tom as "Canadian". Please change to "Texan" or "a graphic designer from Texas" and "the graphic designer from Texas in our company".

https://github.com/freeCodeCamp/freeCodeCamp/pull/52472/files#diff-b09bd864cd310e0f4686bd36c93d39eab8d6c28c912d350021d7c06c813baf40

First Day at the Office - Task 40

Could [place of origin] be out of the backticks in the the description? Also, it shows in round brackets here instead of square, as in other examples. The audio reference and the sentence say "Canada". Please change to "Texas".

https://github.com/freeCodeCamp/freeCodeCamp/pull/52472/files#diff-15e85723012c08863e19843dc7b41437a77b1e4401cb406467b68d148b7b461e

First Day at the Office - Task 41

The audio reference says "Canada", but this should go away in the final version. No need to change, I suppose.

https://github.com/freeCodeCamp/freeCodeCamp/pull/52472/files#diff-bdf297a8d296019f2adc6292ad6b23047ef0b52a7aec5b8c33ec8d793187b6f3

First Day at the Office - Task 42

The sentence says "Canada". Please change to "Texas".

https://github.com/freeCodeCamp/freeCodeCamp/pull/52472/files#diff-03b6d1063d74b4b5503370784dff5a303765a0704520384d5377017cee80f16e

First Day at the Office - Task 51

Please change "They" to "These" in the audio reference and sentence.

https://github.com/freeCodeCamp/freeCodeCamp/pull/52472/files#diff-c501e12a13f1280fcb38f6a1375122ef0f294bc0786e47df1045e135a758b845

First Day at the Office - Task 52

Audio reference says "They" instead of "These", but this should go away in the final version. No need to change, I suppose.

https://github.com/freeCodeCamp/freeCodeCamp/pull/52472/files#diff-65de1b9597a62e8efb31f124af45864b26dbbdde7eacdda3801051e9079d98ad

First Day at the Office - Task 55

@nieldakarla, using the plural for "things" in the description might confuse the learner, as we are explaining "this" and "that" here. How about "something close" and "something a bit far", instead?

https://github.com/freeCodeCamp/freeCodeCamp/pull/52472/files#diff-29fb29ed77a7c2fb3277ad2d01be0ce5d698f0f355c4ad3132cc22097830ad21

First Day at the Office - Task 61

@nieldakarla, technically, "your" and "my" are not "pronouns", but "possessive adjectives", as they do not stand on their own, like pronouns do. Examples of possessive pronouns would be "mine", "yours" etc. Simple suggestion: replace "Pronouns" with "Words" in the beginning of the description.

https://github.com/freeCodeCamp/freeCodeCamp/pull/52472/files#diff-4bc72bfb59bce327962bc0e1722287b49796603039c6084a148c19bd772a768a

First Day at the Office - Task 61

Use lowercase in "standing" in both the sentence and the audio reference.

https://github.com/freeCodeCamp/freeCodeCamp/pull/52472/files#diff-10ce77bb1fcfc0b40a2f8718d3907699f5e042a32122bc1ad405c71e7c929561

First Day at the Office - Task 94

I think "breakfast", "lunch", and "dinner" should go in backticks in the description and in the feedbacks. 

https://github.com/freeCodeCamp/freeCodeCamp/pull/52472/files#diff-21820af4c963353c521c499203a3a3d88631d58abdf45122578f45a6af61e104

First Day at the Office - Task 106

I believe the expression should be "places around here", both in the description and text.

https://github.com/freeCodeCamp/freeCodeCamp/pull/52472/files#diff-fa851dd54fb337aef65719df606ceab41b12027263e55dd409dc96697df16bb0

First Day at the Office - Task 107

@nieldakarla, this fill in the blank exercise has no description. Should it be like this?

https://github.com/freeCodeCamp/freeCodeCamp/pull/52472/files#diff-92f3ff02f3fec8d22ea9ea11f8e51db538fa728276b217516ba797625e53dea8

First Day at the Office - Task 123

In the description, I'd suggest replacing "isn't" with "is not" as in the audio and putting it in backticks to clarify this is what we are trying to explain here.

https://github.com/freeCodeCamp/freeCodeCamp/pull/52472/files#diff-3827f643bd8e0df91da8306832ce08f7121e224c8e5db4469cbc194321491e21

First Day at the Office - Task 126

As the answers are in English, we could leave the question in English as well, with backticks, thus.

https://github.com/freeCodeCamp/freeCodeCamp/pull/52472/files#diff-d7e5de33f588dc30c31ecd18c199f1300308288f9aaa81784c5b807911987579

First Day at the Office - Task 132

Could "[Name]" be removed from backticks here?


https://github.com/freeCodeCamp/freeCodeCamp/pull/52472/files#diff-0615fc2f2a0c9660da0af90ef810e5ce40a4a948176b3cbb5d688aea23b53da1

First Day at the Office - Task 133

Could "[Name]" be removed from backticks here?

https://github.com/freeCodeCamp/freeCodeCamp/pull/52472/files#diff-3f81101fb7f713b53903b5b14ed3f93cfdb4fa51a7255d57081540fdffd2be33

First Day at the Office - Task 135

Sentence: not "Canada", but "Texas".

https://github.com/freeCodeCamp/freeCodeCamp/pull/52472/files#diff-ad931196eab41276bb02044c784b327ef1eb4b57282d4ec218c04b73d3755c94

First Day at the Office - Task 149

Typo: it's "Sophie", not "Sophia".

https://github.com/freeCodeCamp/freeCodeCamp/pull/52472/files#diff-1696703dd892a524605631a9f2a12355f5eeffba265043ebfdeb2ed1932d725b

First Day at the Office - Task 150

Please replace "with the phrase" in the description with "saying". Also, @nieldakarla, how about having this question in English (backticks)?

https://github.com/freeCodeCamp/freeCodeCamp/pull/52472/files#diff-26628e37b5fb86783eb393a7c37d12597e4997f867391dbed02e9417511acd1d

First Day at the Office - Task 163

Two issues in the answer "Jake is not sure, but she thinks." Change "she" to "he" and maybe rephrase: "Jake is not sure. He thinks what he is saying is true."

File: curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656caddea3aba304c6172571.md [hunk 16]
```diff
@@ -0,0 +1,38 @@
...
+
+# --description--
+
+`Ergonomic` refers to the design of products intended to provide optimum comfort and to avoid stress or injury.
+
+# --fillInTheBlank--
+
+## --sentence--
+
+`A Standing Desk, an _ chair and an _ mouse.`
```
Comment (DanielRosa74) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656caddea3aba304c6172571.md:22]: Use lowercase in "standing" in both the sentence and the audio reference.

File: curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656cc3b856f1d01011bfc027.md [hunk 17]
```diff
@@ -0,0 +1,53 @@
...
+audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
+---
+
+<!--
+AUDIO REFERENCE:
+Tom: Wow, I'm so hungry. Is it lunch time?
+-->
+
+# --description--
+In English, the three main meals of the day are breakfast, lunch, and dinner. Breakfast is typically eaten in the morning, lunch is around midday, and dinner is in the evening.
```
Comment (DanielRosa74) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656cc3b856f1d01011bfc027.md:15]: I think "breakfast", "lunch", and "dinner" should go in backticks in the description and in the feedbacks.

File: curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656cc92ecdebaa1278505e87.md [hunk 18]
```diff
@@ -0,0 +1,54 @@
...
+---
+
+<!--
+AUDIO REFERENCE:
+Sophie: Today, I'm going out. I can show you some places around here. Are you interested?
+-->
+
+# --description--
+
+`Places around` refers to locations in the nearby vicinity or area.
```
Comment (DanielRosa74) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656cc92ecdebaa1278505e87.md:16]: I believe the expression should be "places around here", both in the description and text.

File: curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656cc98cfe781112ace7d3ee.md [hunk 19]
```diff
@@ -0,0 +1,44 @@
...
+audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
+---
+
+<!--
+AUDIO REFERENCE:
+Sophie: Today, I'm going out. I can show you some places around here. Are you interested?
+-->
+
+# --description--
+
```
Comment (DanielRosa74) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656cc98cfe781112ace7d3ee.md:15]: @nieldakarla, this fill in the blank exercise has no description. Should it be like this? Suggestion: "Now let's use the expression we learned for referring to nearby places."

File: curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656cd1ca72b3e615a33c05b6.md [hunk 20]
```diff
@@ -0,0 +1,46 @@
...
+---
+
+<!--
+AUDIO REFERENCE:
+Sophie: Well, it is not far.
+-->
+
+# --description--
+
+In English, you can use a certain structure to indicate that something isn't in a particular state or condition in the present.
```
Comment (DanielRosa74) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656cd1ca72b3e615a33c05b6.md:16]: In the description, I'd suggest replacing "isn't" with "is not" as in the audio and putting it in backticks to clarify this is what we are trying to explain here.

File: curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656cd31045ce74162adef6c7.md [hunk 21]
```diff
@@ -0,0 +1,58 @@
...
+
+# --description-- 
+
+This is the last task for this dialogue. Listen to the whole conversation again and answer the question below.
+
+# --question--
+
+## --text--
+
+What are Tom and Sophie planning to do?
```
Comment (DanielRosa74) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656cd31045ce74162adef6c7.md:22]: As the answers are in English, we could leave the question in English as well, with backticks, thus.

File: curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656cd83ad397f61ce1259ad6.md [hunk 22]
```diff
@@ -0,0 +1,30 @@
...
+---
+
+<!--
+AUDIO REFERENCE:
+Sophie: You, too. Brian, let me introduce you to Tom.
+-->
+
+# --description--
+
+When you want to present one person to another, you can use the phrase `let me introduce you to [Name]`. For example: `Let me introduce you to Daniel.`
```
Comment (DanielRosa74) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656cd83ad397f61ce1259ad6.md:16]: Could "[Name]" be removed from backticks here?

File: curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656cd8d68948d11d201308d4.md [hunk 23]
```diff
@@ -0,0 +1,60 @@
...
+---
+
+<!--
+AUDIO REFERENCE:
+Sophie: You, too. Brian, let me introduce you to Tom. Tom, this is Brian.
+-->
+
+# --description--
+
+The phrase `this is [Name]` is used to present someone in a conversation. It's a simple and direct way to make introductions. For example: `Daniel, this is Carol.`
```
Comment (DanielRosa74) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656cd8d68948d11d201308d4.md:16]: Could "[Name]" be removed from backticks here?

File: curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656cd9f4e30e6d1d81f0e62d.md [hunk 24]
```diff
@@ -0,0 +1,33 @@
...
+
+# --description--
+
+The verb `be` has different forms based on the subject. For third person singular (`he`, `she`, `it`), the correct form is `is`.
+
+# --fillInTheBlank--
+
+## --sentence--
+
+`This is Brian. He _ a web developer. This is Tom. He _ from Canada.`
```
Comment (DanielRosa74) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656cd9f4e30e6d1d81f0e62d.md:17]: Sentence: not "Canada", but "Texas".

File: curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656d1d1f46bc820629aecc0d.md [hunk 25]
```diff
@@ -0,0 +1,30 @@
...
+---
+
+<!--
+AUDIO REFERENCE:
+Brian: But it is true. She's the person to go to if you need help, Tom!
+-->
+
+--description--
+
+Watch the dialogue below and listen to how Brian refers to Sophia.
```
Comment (DanielRosa74) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656d1d1f46bc820629aecc0d.md:16]: Typo: it's "Sophie", not "Sophia".

File: curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656d25e0c5d5aa11ade33754.md [hunk 26]
```diff
@@ -0,0 +1,49 @@
...
+id: 656d25e0c5d5aa11ade33754
+title: Task 150
+challengeType: 19
+dashedName: task-150
+audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
+---
+
+# --description--
+
+In the dialogue, Jake introduces himself to Sarah with the phrase `I'm Jake, from Security.` The word `from` is a preposition that shows the origin or source. In this context, `from Security` tells which department Jake works.
```
Comment (DanielRosa74) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656d25e0c5d5aa11ade33754.md:11]: Please replace "with the phrase" in the description with "saying". Also, @nieldakarla, how about having this question in English (backticks)?

File: curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656d2c2fedbd5f14055b7e7b.md [hunk 27]
```diff
@@ -0,0 +1,58 @@
...
+
+`Jake is absolutely certain.`
+
+### --feedback--
+
+`Guess` means there's some uncertainty.
+
+---
+
+`Jake is not sure, but she thinks.`
```
Comment (DanielRosa74) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656d2c2fedbd5f14055b7e7b.md:34]: Two issues in the answer "Jake is not sure, but she thinks." Change "she" to "he" and maybe rephrase: "Jake is not sure. He thinks what he is saying is true."

**Review by DanielRosa74 — COMMENTED** [review]
Overall, I believe the backticks are correct. Added comments to moT01 and nieldakarla where I found something to change/discuss about.

**Review by DanielRosa74 — APPROVED** [review]
LGTM

**Review by naomi-lgbt — APPROVED** [review]

**Review by DanielRosa74 — APPROVED** [review]
LGTM

**Comment by moT01** [comment]
Thanks for all the reviews and getting this in everyone 🎉
